README.md

```diff
@@ -25,18 +25,19 @@ Before contributing, please read through the [Contribution Guide](https://github
 
 ## How do I contribute?
 
-Contributing is as simple as:
+Thanks for your interest; we'd love to work with you! 💖
+Contributing follows this flow:
 
 1. Forking this repo.
 2. Editing your fork.
     - Be sure to fetch the main branch shortly before you make your edits—this repo is constantly changing, and trying to merge too old of a fork can be a challenge!
 4. Making a pull request to merge your fork with this repo.
 
-If you haven't gone through this workflow before, you can check out [this YouTube video](https://www.youtube.com/watch?v=rgbCcBNZcdQ) to learn about how to make a pull request from a fork using Git. Alternatively, if you'd prefer to keep things to the GitHub UI for now, you can follow the instructions in that video up to 1:18 to fork this repo. After that, you can create your entry in your fork using the UI and then make a pull request by pressing this handy button:<br>
+If you haven't gone through this workflow before, you can check out <a alt="YouTube: Creating a Simple Github Pull Request" href="https://www.youtube.com/watch?v=rgbCcBNZcdQ">this YouTube video</a> to learn about how to make a pull request from a fork using Git. Alternatively, if you'd prefer to keep things to the GitHub UI for now, you can follow the instructions in that video up to 1:18 to fork this repo. After that, you can create your entry in your fork using the UI and then make a pull request by pressing this handy button:<br>
```

Comment:
Wait, is there a difference between the `alt` attribute using the `<img>` element and a title link using Markdown?

Link Title: https://www.codecademy.com/resources/docs/markdown/links

Reply:
@JoshuaKGoldberg

Reply:
The difference here is that both links had the same, relatively uninformative link title. Someone receiving a list of all links in the readme would get something like:

* "Codecademy Docs"
* "Contribution Guide"
* "this YouTube video"
* "this YouTube video"

The first two are clear, but the last two aren't. https://www.w3.org/TR/WCAG20-TECHS/H33.html

Ironically my explanation wasn't very clear: thanks for asking! 😄 

> a difference between the `alt` attribute ...

D'oh, I wasn't thinking; I should be using `title`! (per the w3 doc)

Reply:
Dope! I think we can also do this:

`[link text](URL "link title")`

Anyway, merging!

Reply:
TIL!